### PR TITLE
Muflus mechanical pass

### PR DESCRIPTION
**Pull request by manucamejo**
Closes some bullets of #644 

To test it, open the issue and test each checked bullet.

### CODE REVIEW

File: client/Assets/StreamingAssets/Skills.json [hunk 1]
```diff
@@ -148,13 +148,13 @@
       "DoFunc": "0",
       "ButtonType": "1",
       "Cooldown": "1000",
-      "Damage": "15",
+      "Damage": "50",
```
[the remark this answers is not in the document]
Comment (manucamejo) [reply client/Assets/StreamingAssets/Skills.json]: Thanks

File: server/native/gamestate/src/game.rs [hunk 2]
```diff
@@ -939,6 +1019,23 @@ impl GameState {
                 );
                 Ok(Vec::new())
             }
+            Name::Muflus => {
+                attacking_player.add_effect(
+                    Effect::FieryRampage.clone(),
```
Comment (tvillegas98) [inline server/native/gamestate/src/game.rs:1024]: not needed to clone

File: server/native/gamestate/src/game.rs [hunk 3]
```diff
@@ -1193,6 +1308,39 @@ impl GameState {
...
+        self.players.iter_mut().for_each(|player| {
+            if matches!(player.status, Status::DEAD) {
+                return;
+            }
+            let mut effect_data = match player.effects.get(&Effect::Burned) {
+                Some(data) => data.clone(),
+                None => return,
+            };
+
+            let delta = (1000 / 2) as u32;
```
Comment (tvillegas98) [inline server/native/gamestate/src/game.rs:1323]: This variable intends to be parameter? if not, `delta = 500`

File: server/native/gamestate/src/utils.rs [hunk 4]
```diff
@@ -24,3 +24,16 @@ pub fn cmp_float(f1: f64, f2: f64) -> Ordering {
         Ordering::Equal
     }
 }
+
+pub fn angle_between_vectors(v1: RelativePosition, v2: RelativePosition) -> u64 {
+    let angle1 = (v1.y as f32).atan2(v1.x as f32).to_degrees();
+    let angle2 = (v2.y as f32).atan2(v2.x as f32).to_degrees();
+
+    let mut a = angle1 - angle2;
```
Comment (tvillegas98) [inline server/native/gamestate/src/utils.rs:32]:
```suggestion
    let mut angle_diff = angle1 - angle2;
```

File: server/native/gamestate/src/game.rs (hunk 3 above)
Comment (manucamejo) [reply server/native/gamestate/src/game.rs:1323]: yes, it should be a config

**Pull request merged**